modules/books-manager.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable radix */
+
```

Comment:
- [OPTIONAL] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 😅

Reply:
Fixed